CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 Changelog
 ===
+## [6.6.0](https://github.com/mwz/sonar-scala/releases/tag/v6.6.0) - ??.09.2018
+- Refactored Scalastyle module and introduced new Scalastyle rules repository along with a new quality profile, which now in addition to the latest Scalastyle rules also consists of rule templates which are activated by default. The existing Scalastyle quality profile and rule registry have been deprecated and will be removed in the next major version of the plugin (7.x). The new module also exposes the following optional property `sonar.scala.scalastyle.disable`, which allows you to disable the Scalastyle sensor. (#35 - @mwz)
+- New `Scalastyle+Scapegoat` quality profile, which includes all of the default rules from Scalastyle and Scapegoat quality profiles. (#112 - @BalmungSan)
+- Bumped Scapegoat up to `1.3.7`, which introduces a new inspection ([UnsafeStringContains](https://github.com/sksamuel/scapegoat/blob/v1.3.7/src/main/scala/com/sksamuel/scapegoat/inspections/string/UnsafeStringContains.scala)) and other various fixes and improvements - see the [diff](https://github.com/sksamuel/scapegoat/compare/v1.3.5...v1.3.7) for more details. (#100 - @mwz)
```

Comment:
I think we should do a general dumping of dependencies before the release, especially **classgraph** _(see [this](https://github.com/mwz/sonar-scala/pull/110#discussion_r217233297) comment)_, and **Scapegoat** to [`1.3.8`](https://github.com/sksamuel/scapegoat/releases/tag/v1.3.8).

Reply:
Scapegoat 1.3.8 doesn't add anything that would be of interest to us so I'll skip it for now and I'll bump up the rest in a separate PR.

Reply:
Ok